relay/kusama/src/lib.rs

```diff
@@ -684,6 +683,8 @@ impl pallet_staking::Config for Runtime {
 	type NominationsQuota = pallet_staking::FixedNominationsQuota<{ MaxNominations::get() }>;
 	type MaxUnlockingChunks = frame_support::traits::ConstU32<32>;
 	type HistoryDepth = frame_support::traits::ConstU32<84>;
+	// TODO:(PR#159) check/set correct constant and remove this comment!
+	type MaxControllersInDeprecationBatch = ConstU32<751>;
```

Comment:
There are 5596 unmatching stash / controller pairs on Kusama remaining.

```suggestion
	type MaxControllersInDeprecationBatch = ConstU32<5596>;
```